miniscope_io/models/stream.py

```diff
@@ -68,10 +71,18 @@ class StreamDaqConfig(MiniscopeConfig, YAMLMixin):
         If `LSB`, then the incoming bitstream is in the format that each 32-bit words are bit-wise reversed on its own.
         Furthermore, the order of 32-bit words in the pixel data within the buffer is reversed (but the order of words in the header is preserved).
         Note that this format does not correspond to the usual LSB-first convention and the parameter name is chosen for the lack of better words.
+    save_video : bool, optional
+        Whether the video gets saved as an .avi file, by default False.
+    save_video_name : str, optional
+        If video is saved, , by default False.
+    show_video : bool, optional
+        Whether the video is showed in "real-time", by default True.
 
     ..todo::
         Takuya - double-check the definitions around blocks and buffers in the firmware and add description.
     """
+
+    mode: Optional[str] = 'DAQ'
```

Comment:
`Optional` is equivalent to `T | None`. since `None` is not a valid value here, this should just be a `str` with a default value.

or, if it is one of the `ALLOWED_MODES` it should be something like `Literal[*ALLOWED_MODES]`